src/ui/public/state_management/state.js

```diff
@@ -18,14 +18,14 @@ export default function StateProvider(Private, $rootScope, $location, config) {
   const Events = Private(EventsProvider);
 
   _.class(State).inherits(Events);
-  function State(urlParam, defaults, { hashingStore, notify } = {}) {
+  function State(urlParam, defaults, { hashingStore, notifier } = {}) {
     State.Super.call(this);
 
     let self = this;
     self.setDefaults(defaults);
     self._urlParam = urlParam || '_s';
-    this._notify = notify || new Notifier();
-    self._hasher = hashingStore || new HashingStore({
+    this._notifier = notifier || new Notifier();
+    self._hashingStore = hashingStore || new HashingStore({
```

Comment:
Same ideas here -- use a lower-cased version of the class name for the instance name.